packages/static-build/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "allowJs": true,
```

Comment:
Why did this setting change?

Reply:
I changed it in `next` and `node` builders also. It was added because `ts-jest` logs a warning:

```
ts-jest[ts-jest-transformer] (WARN) Got a `.js` file to compile while `allowJS` option is not set to `true`
```

The warning says to enable `allowJs: true` to fix it. In hindsight, this probably should have been added to the `test/tsconfig.json` file instead of the package-level, but I didn't see any side effects of it being in the package-level `tsconfig.json`.

Do you think I should move them?

Reply:
@styfle Just in case :) https://github.com/vercel/vercel/pull/9862